benchmark/benchmark_test.go

```diff
@@ -65,10 +74,9 @@ func init() {
 						_, err := io.Copy(buf, str)
 						Expect(err).NotTo(HaveOccurred())
 					})
-					// this is *a lot* faster than Expect(buf.Bytes()).To(Equal(data))
-					Expect(bytes.Equal(buf.Bytes(), data)).To(BeTrue())
+					Expect(buf.Bytes()).To(Equal(data))
 
-					b.RecordValue("transfer rate [MB/s]", float64(dataLen)/1e6/runtime.Seconds())
+					b.RecordValue("transfer rate [MB/s]", float64(dataLen)/(1<<20)/runtime.Seconds())
```

Comment:
If you really want Mebibytes, please adjust the unit accordingly ;)

Reply:
I'll replace the 1<<20 by 1e6.